### DISCUSSION TITLE
..

### DISCUSSION THREAD

**Discussion by bitplorer** [tailwindlabs/tailwindcss#5417] [Help]
...

**Comment by JonathonOsinski** [comment]
Hey, We have a small website, and we use bootstrap. The css and js files are the main cause of slowing down, since they are huge.

How can we delete unnecessary css classes inside .css files?


[greatpeople me](https://www.greatpeople-me.org/)

**Comment by CFarhad** [comment]
Hi, you should use [Purgecss](https://purgecss.com/) for this problem

if it was helpful please rate up
